person.rb

```diff
@@ -0,0 +1,42 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(name: 'Unknown', age: 0, parent_permission: true)
+    @id = rand(1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def getter_id
+    @id
+  end
```

Comment:
- [ ] [OPTIONAL] great job creating the getter for `@id` here 🥇 if you would like to make your project look even better and cleaner you might want to consider using the shortcuts as explained in [this lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor)